member-service/src/main/java/com/hedvig/memberservice/web/QualityAssuranceController.kt

```diff
@@ -0,0 +1,25 @@
+package com.hedvig.memberservice.web
+
+import com.hedvig.memberservice.services.QualityAssuranceServiceImpl
+import com.hedvig.memberservice.web.dto.UnsignMemberRequest
+import org.springframework.context.annotation.Profile
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@Profile("staging", "development")
+@RequestMapping("/_/staging/")
+class QualityAssuranceController(
+    private val qualityAssuranceService: QualityAssuranceServiceImpl
+) {
+    @PostMapping("{country}/member/unsign")
```

Comment:
Could be since it's one action
```suggestion
    @PostMapping("{country}/unsignMember")
```